resources/todomvc/architecture-examples/vue/src/components/TodosComponent.vue

```diff
@@ -61,50 +59,48 @@ export default {
         }
     },
     methods: {
-        addTodo() {
+        addTodo(value) {
             this.todos.push({
                 completed: false,
-                title: this.newTodo
-            })
-            this.newTodo = ''
+                title: value,
+                id: uuid()
```

Comment:
Am I right that the previous version had no `id` value despite that we were refering to it in the loop?
(nothing to do here, thanks for the change!)